package/kedro_viz/server.py

```diff
@@ -320,15 +321,16 @@ def _call_viz(
     global data  # pylint: disable=global-statement,invalid-name
 
     if load_file:
-        import logging
         # Remove all handlers for root logger
         root_logger = logging.getLogger()
         root_logger.handlers = []
 
         data = _load_from_file(load_file)
     else:
         if match(kedro.__version__, ">=0.15.0"):
-            from kedro.context import KedroContextError
+            from kedro.context import (  # pylint: disable=import-error,import-outside-toplevel
```

Comment:
Why does it complain about import error?

Reply:
Oops, sorry it was my copy/paste error.. will remove it.